packages/fxa-settings/src/components/ThirdPartyAuth/index.tsx

```diff
@@ -11,31 +11,26 @@ import { ReactComponent as AppleLogo } from './apple-logo.svg';
 import { useAccount, useConfig } from '../../models';
 import Storage from '../../lib/storage';
 import { AUTH_PROVIDER } from 'fxa-auth-client/browser';
+import { ReactElement } from 'react-markdown/lib/react-markdown';
 
 export type ThirdPartyAuthProps = {
   enabled?: boolean;
+  onContinueWithGoogle?: FormEventHandler<HTMLFormElement>;
+  onContinueWithApple?: FormEventHandler<HTMLFormElement>;
 };
 
-type ThirdPartyAuthSigninParams = {
-  authorizationEndpoint: string;
-  clientId: string;
-  scope: string;
-  redirectUri: string;
-  responseType: string;
-  responseMode?: string;
-};
-
-const GOOGLE_SCOPES = 'openid email profile';
-const APPLE_SCOPES = 'email';
-
 /**
  * ThirdPartyAuth component
  * A React component that renders Google and Apple third-party authentication buttons.
  * It handles user sign-in with the respective provider when the buttons are clicked.
  */
-const ThirdPartyAuth = ({ enabled = false }: ThirdPartyAuthProps) => {
-  const config = useConfig();
+const ThirdPartyAuth = ({
+  enabled = false,
+  onContinueWithGoogle,
+  onContinueWithApple,
```

Comment:
We could use another prop to conditionally display the separator. This was added since this component was created, in content-server it's called `isSignup` but something like `showSeparator` is probably better.